components/raftstore/src/store/worker/read.rs

```diff
@@ -373,7 +375,7 @@ where
             cache_read_id,
             store_id: Cell::new(None),
             metrics: Default::default(),
-            delegates: HashMap::default(),
+            delegates: LruCache::with_capacity(DELEGATE_LRU_CACHE_CAPACITY),
```

Comment:
I suggest to use default. There can be no allocation at all for learners.

Reply:
There is not `LruCache::default`,  underneath `LruCache::with_capacity` using `HashMap::default` to initialize, so it won't allocate large memory until it needs to.